modules/appeals_api/spec/models/higher_level_review_spec.rb

```diff
@@ -114,52 +115,10 @@
     it('matches header') { is_expected.to eq auth_headers['X-VA-Insurance-Policy-Number'] }
   end
 
-  describe '#number_and_street' do
```

Comment:
Since these are shared methods between v1 and v2, we should keep them in this area and not move them into  a v2 context.